### PR TITLE
Add check for some requests if SSL was set 

### PR DESCRIPTION
**Pull request by mmizin**

### CODE REVIEW

**Review by SergeyMoroz0703 — APPROVED** [review]

File: app/jmeter/jira.jmx [hunk 1]
```diff
@@ -6500,62 +6514,69 @@ if ( sleep_time &gt; 0 ) {
...
-                  <stringProp name="HTTPSampler.method">GET</stringProp>
-                  <boolProp name="HTTPSampler.follow_redirects">true</boolProp>
-                  <boolProp name="HTTPSampler.auto_redirects">false</boolProp>
-                  <boolProp name="HTTPSampler.use_keepalive">true</boolProp>
-                  <boolProp name="HTTPSampler.DO_MULTIPART_POST">false</boolProp>
-                  <stringProp name="HTTPSampler.embedded_url_re"></stringProp>
-                  <stringProp name="HTTPSampler.connect_timeout"></stringProp>
-                  <stringProp name="HTTPSampler.response_timeout"></stringProp>
-                </HTTPSamplerProxy>
+                <IfController guiclass="IfControllerPanel" testclass="IfController" testname="if ssl not setuped" enabled="true">
```
Comment (ometelytsia) [inline app/jmeter/jira.jmx:6517]: fix wording

File: app/jmeter/jira.jmx [hunk 2]
```diff
@@ -7239,62 +7260,69 @@ if ( sleep_time &gt; 0 ) {
...
```
Comment (ometelytsia) [inline app/jmeter/jira.jmx:7263]: fix wording

File: app/jmeter/jira.jmx [hunk 3]
```diff
@@ -8098,62 +8126,69 @@ if ( sleep_time &gt; 0 ) {
...
```
Comment (ometelytsia) [inline app/jmeter/jira.jmx:8129]: Fix wording.

File: app/jmeter/jira.jmx [hunk 4]
```diff
@@ -9616,7 +9651,7 @@ if ( sleep_time &gt; 0 ) {
             <connectTime>true</connectTime>
           </value>
         </objProp>
-        <stringProp name="filename"></stringProp>
+        <stringProp name="filename">/Users/mmizin/Projects/dc-app-performance-toolkit/app/results/jira/2021-04-23_09-32-59/error.jtl</stringProp>
```
Comment (ometelytsia) [inline app/jmeter/jira.jmx:9654]: Remove this.

**Pull request closed without merging**